### ISSUE TITLE
Language server doesn't handle `css.resolve`

### ISSUE THREAD

**Issue by SMores** [Divlo/vscode-styled-jsx#58]
Hello! I see that a previous issue (#16) called out a bug when a component uses `css.global`; I'm seeing what seems to be the same issue when using `css.resolve`:

<img width="266" alt="image" src="https://user-images.githubusercontent.com/5354254/113458443-45ec3f00-93e0-11eb-90b1-b32b274bb520.png">

## Steps To Reproduce

1. Add a tagged template using `css.resolve`

## The current behavior

Syntax highlighting and language server responses (autocomplete options) are incorrect/partially missing

## The expected behavior

`css.resolve`-tagged templates behave the same as `<styled jsx>` tags and `css.global`-tagged-templates!

**Comment by Divlo** [comment]
Thanks for your report! @SMores 
I don't know if it should be the case, but for your burgerStyles to work you need to wrap it under a className like so :
```js
import css from 'styled-jsx/css'

const burgerStyles = css.resolve`
  .some-class-name {
    display: block;
    position: absolute;
    top: 2.5rem;
    right: 2rem;
  }

  @media (min-width: 769px) {
    .some-class-name {
      display: none;
    }
  }
`
````

**Comment by SMores** [comment]
Interesting! I wonder if this is a quirk of how css.resolve works: it _does_ allow styles at the top level (without a tag)... but now that I'm looking more closely at the docs, this isn't actually a documented feature. This works just fine as is:

<img width="169" alt="image" src="https://user-images.githubusercontent.com/5354254/113484623-45eb4e00-9477-11eb-9f5d-7a08c87e7397.png">

But you're right that the docs suggest nesting these styles in a tag... I personally think that kind of stinks/breaks encapsulation, but I suppose it's fair that your extension doesn't support it given that it isn't documented!

I just opened a thread on the styled-jsx discussion forum asking for clarification: https://github.com/vercel/vercel/discussions/6065.

**Comment by eliasthompson** [comment]
Related to the topic but not related to the issue of wrapping resolved styles in a className: In our code base we're destructuring the `resolve` import from styled-jsx/css (`import { resolve } from 'styled-jsx/css';`), which results in us declaring:
```javascript
resolve`
   ...
`
```
This doesn't trigger the syntax highlighting as it specifically looks for `css.resolve`. Are there any plans to support this either by default or as a configurable option?

(If you feel this should be a separate issue just lmk.)

**Comment by Divlo** [comment]
Right, I think it is cleaner to use `css.resolve`, but yes feel free to open a PR with a configurable option to make it work with `resolve` only. @eliasthompson

**Comment by SMores** [comment]
I never ended up getting any responses to that discussion thread.. just tried another tack and opened an issue on styled-jsx!
